rs/sns/governance/src/pb/conversions.rs

```diff
@@ -79,14 +79,21 @@ impl From<pb_api::DisburseMaturityInProgress> for pb::DisburseMaturityInProgress
 
 impl From<pb::Neuron> for pb_api::Neuron {
     fn from(item: pb::Neuron) -> Self {
-        let topic_followees = Some(
-            item.topic_followees
-                .into_iter()
-                .map(|(topic, followees)| {
-                    let followees = pb_api::neuron::Followees::from(followees);
-                    (topic, followees)
-                })
-                .collect(),
+        let topic_followees = item.topic_followees.map(
+            |pb::neuron::TopicFollowees {
+                 topic_id_to_followees,
+             }| {
```

Comment:
I think we almost never destructure in the parameter list. Instead, we do that in the body. I.e.

```suggestion
            |topic_followees| {
                let pb::neuron::TopicFollowees {
                    topic_id_to_followees,
                } = topic_followees;

```

I see no particular reason to be inconsistent here. It only saves one line.

Reply:
Could you explain which convention you are referring to?

Reply:
> we almost never destructure in the parameter list.

I think Rust doesn't allow this in regular functions, only lambdas. 

To be honest, I don't see a disadvantage.

Reply:
Moreover, after thinking about it for a few moments, I actually prefer to deconstruct the type right away, since that's the intention of the function — to consume the whole thing, converting it to a different representation. Giving it a name seems redundant.